src/server/services/status.js

```diff
@@ -11,13 +11,25 @@ export async function getAllStatus() {
 }
 
 /**
+ * Fetch a service by its name.
+ *
+ * @param  {String} id
+ * @return {Promise}
+ */
+export async function getStatusGroupedByName() {
+  return await new Status().query(qb => qb.groupBy('name')
```

Comment:
I would prefer querybuilder until and unless we absolutely need write raw queries. We are already using a library and we need to make use of its features as much as we can. Raw queries will obviously be readable but querybuilder is one of the powerful feature of knex (which is also very well documented). What we can do is have a same standard for using query builder throughout the code.